docs/memcached.md

```diff
@@ -0,0 +1,54 @@
+# `memcached` Metrics Receiver
+
+The memcached receiver can retrieve stats from your memcached server through the . 
+
+
+## Configuration
+
+Following the guide for [Configuring the Ops Agent](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/configuration#file-location), add the required elements for your memcached configuration.
+
+To configure a receiver for your memcached metrics, specify the following fields:
+
+| Field                 | Default                   | Description |
+| ---                   | ---                       | ---         |
+| `type`                | required                  | Must be `memcached`. |
+| `endpoint`            | `localhost:3306`          | The url, or unix socket file path, for your memcached server. |
+| `collection_interval` | `60s`                     | A [time.Duration](https://pkg.go.dev/time#ParseDuration) value, such as `30s` or `5m`. |
+
+Example Configuration:
+
+```yaml
+metrics:
+  receivers:
+    redis_metrics:
+      type: memcached
+      address: localhost:3306
+      collection_interval: 30s
+  service:
+    pipelines:
+      memcached_pipeline:
+        receivers:
+          - memcached_metrics
+```
+
+## Metrics
+
+The Ops Agent collects the following metrics from your memcached servers.
+
+| Name | Description | Unit | Type | Attributes |
+| ---- | ----------- | ---- | ---- | ---------- |
+| memcached.bytes | Current number of bytes used by this server to store items. | By | Gauge | <ul> </ul> |
```

Comment:
This was an existing metric on the otel receiver. the memcached docs describe it as [`Current number of bytes used by this server to store items.` ](https://docs.oracle.com/cd/E17952_01/mysql-5.6-en/ha-memcached-stats-general.html). Do you have a suggestion for better name? perhaps `memcached.storage`?

Reply:
I see from the link in your other comment thread that `bytes` is the name that memcached uses for this metric. I think that means we should probably keep it even if it's a little confusing, since that's what users coming from other monitoring systems will expect.